js/views/pageNavVw.js

```diff
@@ -167,11 +175,38 @@ module.exports = Backbone.View.extend({
       this.accWin.css('left', function(){
         return oldPos + moveBy;
       });
+      // switch active tab
+      var curActive = $(this.$el).find('.accordion-active');
+      curActive.addClass('accordion-inactive').removeClass('accordion-active');
+      var newActive = curActive.prev('.accordion-child');
+      newActive.addClass('accordion-active').removeClass('accordion-inactive');
       // focus search input
-      $(this).closest('.accordion-child').prev('.accordion-child').find('.search').focus();
+      newActive.find('.search').focus();
     }
   },
 
+  triggerOnEnterSpace: function(e, cb) {
+    "use strict";
+    switch (e.which) {
+      case 32: // space
```

Comment:
Why trigger this when hitting the space key? Is it for better accessibility? I don't object, I'm just curious as to what your thoughts are.